package.json

```diff
@@ -179,6 +179,7 @@
     "react-test-renderer": "^16.3.2",
     "regenerator-runtime": "^0.13.2",
     "terriajs-server": "^3.0.1",
+    "webpack-cli": "^3.3.11",
```

Comment:
Not sure why this is here?

Reply:
you need it to run
```
  yarn dev
```

Reply:
Huh - well there you go, just did a fresh clone and got the same. 
````
> webpack-dev-server --inline --config buildprocess/webpack.config.dev.js --host 0.0.0.0

The CLI moved into a separate package: webpack-cli
Please install 'webpack-cli' in addition to webpack itself to use the CLI
-> When using npm: npm i -D webpack-cli
-> When using yarn: yarn add -D webpack-cli
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'webpack-cli/bin/config-yargs'
````
I wonder if it's related to a new version of webpack? 
Or if the same issue occurs if you run terriajs as a package within a terriamap instance - in my test I just pulled down terriajs. TerriaMap has webpack-cli as a dependency already so that might circumvent the issues when running it nested as a package.

Reply:
I have never run terriajs as a package within a terriamap instance locally.